app/models/documentation/Config.scala

```diff
@@ -8,6 +8,12 @@ import scala.util.matching.Regex
 
 case class DocumentationConfig(default: TranslationConfig, translations: Seq[TranslationConfig])
 
+case class RedirectPage(from: String, to: String)
```

Comment:
From reading the code, `to` can be either a URL or a new documentation page name. Right?

Reply:
Right.